wire-grpc-client/src/jvmMain/kotlin/com/squareup/wire/GrpcClient.kt

```diff
@@ -81,13 +84,45 @@ actual abstract class GrpcClient actual constructor() {
           for ((key, value) in requestMetadata) {
             addHeader(key, value)
           }
+
+          if (timeout.hasDeadline()) {
+            addHeader("grpc-timeout", serializeTimeout(timeout.deadlineNanoTime()))
```

Comment:
Does the order of the headers matter? The spec says list this one to be present after `te` header. Should I reorder this?

Reply:
I don't think so? The spec only lists the headers in a specific order but doesn't say they have to be sent into this specific one, right?

Reply:
yup, right.